public/api/swagger.json

```diff
@@ -0,0 +1,838 @@
+                "security": [
+                    {
+                        "api_key": []
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Returns a list of auctions",
+                        "schema": {
+                            "type": "object",
+                            "required": [
+                                "auctions"
+                            ],
+                            "properties": {
+                                "auctions": {
+                                    "type": "array",
+                                    "uniqueItems": true,
+                                    "items": {
+                                        "$ref": "#/definitions/Auction"
+                                    }
+                                }
+                            }
+                        },
+                        "examples": {
+                            "application/json": {
+                                "auctions": [
+                                    {
+                                        "issue_url": "https://github.com/18F/mpt3500/issues/10",
+                                        "github_repo": "https://github.com/18F/mpt3500",
+                                        "start_price": 3500,
+                                        "start_datetime": "2016-01-26T18:00:00+00:00",
+                                        "end_datetime": "2016-02-27T17:00:00+00:00",
+                                        "title": "Review a blog post",
+                                        "description": "Quis autem vel eum iure reprehenderit qui in ea voluptate velit esse quam nihil molestiae consequatur, vel illum qui dolorem eum fugiat quo voluptas nulla pariatur?\r\n\r\n## A Markdown Quote\r\n\r\n> Ut enim ad minima veniam, quis nostrum exercitationem ullam corporis suscipit laboriosam, nisi ut aliquid ex ea commodi consequatur? Quis autem vel eum iure reprehenderit qui in ea voluptate velit esse quam nihil molestiae consequatur, vel illum qui dolorem eum fugiat quo voluptas nulla pariatur?\r\n\r\n```\r\ndef foo(bar)\r\n  10.times do\r\n     puts bar\r\n   end\r\nend\r\n```",
+                                        "id": 1,
+                                        "created_at": "2015-12-21T16:40:01+00:00",
+                                        "updated_at": "2015-12-21T16:40:01+00:00",
+                                        "summary": "## Summary\r\n\r\nWe need some prose to be proof-read.",
+                                        "bids": [
+                                            {
+                                                "bidder_id": null,
```

Comment:
This example is taken from Alan's documentation. I think this is a case of redaction in action